src/components/aboutUs/components/constants.js

```diff
@@ -0,0 +1,86 @@
+        marketers and brands do not know where their target users are, how to reach
+        them, or even how much the need to spend to do so.Through out the history of business, 
+        people use data to make more informed decisions. Our goal at Agroeasy, is to make the 
+        istribution of products between producers, marketers and consumers easy.We want to make
+        data / information available to as mamy as possible. `,
+
+    },
+};
+
+export const TEAM_INFO = [
+    {
+        description: `
+            Front-end developer at Agroeasy, he's a graduate of Federal university 
+            of technology, owerri, imo state, Studied Biochemistry (B.Tech). He has great interest
+            in the use of computer and the internet and yes decided to channel that interest
+            to software engineering. His carear as a software engineer started at Enye (2017),
+            where he became aware of numerous challenges that could be solved via the tech.
+            At Enye, he learnt how best to be able to accomplish. His focus and passion will 
+            ensure Agroeasy provides the best of service to all our customers. `,
+        name: "Josh Obinna",
+        photo: '/images/obinna.jpg',
+    },
+    {
+        description: `
+            She loves playing around with codes, encountering errors, fixing bugs, reading codes,
+            interacting and implementing. Being a developer of Agroesy is a basic tool that cemented
+            her foundation in web development. Studied Material & Metallurgical engineering (B.Engr)
+            at the University of Nigeria, Nsuka, Enugu State. For some reason, decided to explore 
+            the world of software development. She's also a lover of graphic design. She looks 
+            forward to building one of the largest Agro E-commerce platform with the collaboration 
+            of her colleagues,`,
+        name: "Chioma Onyekpere",
+        photo: '/images/chiomaa.jpg',
+    },
+    {
+        description:  `
+            He is a dedicated full-statck web developer, who is skilled in javascript.  A talented 
+            software engineer, proficient in react js for frontend and node js for 
+            backend. He is a graduate of information management technology (B.Tech), Federal 
+            University of Technology, Owerri, Imo State. Currently, he is a student at Enye, where 
+            he is becoming the engineer he yearns for, under the mentorship of Uche Nnadi (CEO). 
+            Presently, he is on internship at Silicon jungle labs, where he is  leveraging on the 
+            power of more experts to reach a climax where, he will be one of the best. `, 
+        name: "Okemmadu Eric",
+        photo: '/images/okemmadu.jpg',
+    },
+];
+
+export const MARKET_SCENES = [
+    '/images/rfdtfa.jpg',
+    '/images/oilmill.jpg',
+    '/images/marketer.jpg',
+    '/images/market.jpg',
+    '/images/fishseller.jpg',
+];
```

Comment:
this is a proper syntax for an array with multiple items that can't fit on one line. Can you look through your code and make the other array definitions match this syntax/pattern?